cluster/manifests/skipper/deployment.yaml

```diff
@@ -1,35 +1,57 @@
 {{ $internal_version := "v0.18.42-691" }}
 {{ $version := index (split $internal_version "-") 0 }}
 
+{{ template "skipper-ingress" dict
+  "ConfigItems" .ConfigItems
+  "name" "skipper-ingress"
+  "internal_version" $internal_version
+}}
+
+{{ if .ConfigItems.skipper_ingress_canary_internal_version }}
+{{ template "skipper-ingress" dict
+  "ConfigItems" .ConfigItems
```

Comment:
ConfigItems seem equal in both cases, does it need to be defined here? 🤔

Reply:
Each template invocation receives its own `.` as a context, i.e. there is no global scope

Reply:
> A variable's scope extends to the "end" action of the control structure ("if", "with", or "range") in which it is declared, or to the end of the template if there is no such control structure. A template invocation does not inherit variables from the point of its invocation.